pkgs/applications/graphics/displaycal/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, python3
+, xorg
+, ffmpeg
+, argyllcms
+, gsettings-desktop-schemas
+, gtk3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "displaycal";
+  version = "3.9.10";
+  format = "pyproject";
+
+  src = python3.pkgs.fetchPypi {
+    pname = "DisplayCAL";
+    inherit version;
+    hash = "sha256-oDHDVb0zuAC49yPfmNe7xuFKaA1BRZGr75XwsLqugHs=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    certifi
+    dbus-python
+    build
+    wxPython_4_1
+    send2trash
+    PyChromecast
+    distro
+  ];
+
+  buildInputs = with xorg; [
+    libX11
+    libXxf86vm
+    libXext
+    libXinerama
+    libXrandr
+  ];
+
+  doCheck = false; # Tests try to access an X11 session.
+
+  pythonImportsCheck = [ "DisplayCAL" ];
+
+  # no idea why it looks there - symlink .json lang (everything)
+  postInstall = ''
+    for x in $out/share/DisplayCAL/*; do
+      ln -s $x $out/${python3.sitePackages}/DisplayCAL
+    done
+  '';
+
+  makeWrapperArgs = [
+    "--suffix PATH : ${lib.getBin ffmpeg}"
+    "--prefix PATH : ${lib.makeBinPath [ argyllcms ]}"
+    "--prefix PYTHONPATH : $PYTHONPATH"
+    "--prefix XDG_DATA_DIRS : ${gsettings-desktop-schemas}/share/gsettings-schemas/${gsettings-desktop-schemas.name}:${gtk3}/share/gsettings-schemas/${gtk3.name}"
```

Comment:
Can we use the gappsWrapper instead?

Reply:
Is that the equivalent?

Reply:
Doesn't seem to be the case. It complains about missing schemas and crashes when trying to open file browser.